src/config.ts

```diff
@@ -9,6 +9,7 @@ export const spikeReqBody: object = {
     audience: process.env.AUDIENCE || 'kartoffel',
 };
 export const spikeServiceURL: string = process.env.SPIKE_SERVICE_URL || 'spike-service:8080';
+export const kartoffelQuery: string = `/search?domainusers.datasource=${process.env.DOMAINUSERS_QUERY}`;
```

Comment:
Add a default for DOMAIN_USERS_QUERY.
Also, remember to update the docker compose of each service in the chain upwards (api-gw, web-ui)